### ISSUE TITLE
/etc/environment length limitation to 255 chars

### ISSUE THREAD

**Issue by maczet** [neutrinolabs/xrdp#2705]
I set the access token to /etc/environment 375 chars. 
When I try to read the token from xrdp I get 255 chars.
When I read the token from ssh I get a full token. 

I'm using Ubuntu 22.04.2 LTS and xrdp 0.9.17.

**Comment by matt335672** [comment]
Sorry @maczet - I don't understand what you're doing. Can you add more detail?

**Comment by maczet** [comment]
Hi @matt335672, I will try to explain it better. 
I have access token 375 lengths. When I set token to /etc/environment eg. MY_TOKEN=eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxNjIzOTAyMn0.VFb0qJ1LRg_4ujbZoRMXnVkUgiuKq5KxWqNdbKq_G9Vvz-S1zZa9LPxtHWKa64zDl2ofkT8F6jBt_K4riU-fPgeyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCDkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlh
 and:
I read the token when I logged on the Ubuntu I got the full token 375 chars in length. 
I read the token using xrdp I got part of the token 255 chars in length.
